app.py

```diff
@@ -121,16 +122,38 @@ def ok(message):
 
 
 #### PUBLIC RESOURCES:
-@app.route('/')
-@app.route('/home')
-def home():
-    return render_template('home.html')
+@app.route("/")
+@app.route("/index")
+def index():
+    return '<a href="/exercise_TestLink">Test</a>'
+
+
+@app.route("/exercise_<exercise_link>")
+def exercise(exercise_link):
+    return render_template("exercise.html", exercise_link=exercise_link)
+
+
+@app.route("/check", methods=["POST"])
+def check():
+
+    ANSWERS = {'task1': 'am', 'task2': 'were'}
+    check_answer_in_db = lambda _id, _answer: ANSWERS[_id] == _answer
+
+    # https://javascript.info/bubbling-and-capturing
+    payload = json.loads(request.data.decode('utf-8'))
+    task_id = payload["task_id"]
+    task_answer = payload["task_answer"]
+    print(payload)
+    # TODO remove contraction sensitivity (parse "'m" as "am" and so on)
```

Comment:
we need to think how to deal with it in general - perhaps, the manticore can do it for us